cli/src/diff_util.rs

```diff
@@ -906,7 +907,12 @@ fn diff_content(
             executable: _,
         } => Ok(FileContent {
             is_binary: false,
-            contents: materialize_merge_result_to_bytes(&contents, conflict_marker_style).into(),
+            contents: materialize_merge_result_to_bytes(
+                &contents,
+                conflict_marker_style,
+                choose_materialized_conflict_marker_len(&contents),
```

Comment:
nit: maybe this change is a merge/squash error? I don't think callers usually want to specify the marker length.